app/assets/javascripts/sidebar_toggle.js.erb

```diff
@@ -1,69 +1,69 @@
-(function(global) {
-  'use strict';
-  global.SideBarToggle = (function() {
-    var STORAGE_TOGGLE_KEY = "scinote-sidebar-toggled";
+var SideBarToggle = (function() {
+  var STORAGE_TOGGLE_KEY = "scinote-sidebar-toggled";
+  var WRAPPER = '#wrapper'
+  var SIDEBAR_CONTAINER = ".sidebar-container"
 
-    function show() {
-      $('#wrapper').removeClass('hidden2');
-      $('#wrapper').css('paddingLeft', '280px');
-      $('.navbar-secondary').css(
-        { 'margin-left': '-280px', 'padding-left': '280px' }
-      );
-      $('#sidebar-arrow').attr('data-shown', '');
-      sessionStorage.setItem(STORAGE_TOGGLE_KEY, "un-toggled");
-      $('#wrapper').trigger('sideBar::show');
-      $('#wrapper').one("transitionend", function() {
-        $('#wrapper').trigger('sideBar::shown');
-      })
+  function show() {
+    $(SIDEBAR_CONTAINER).removeClass('collapsed');
+    $(WRAPPER).css('paddingLeft', 'var(--wrapper-width)');
+    $('.navbar-secondary').removeClass("navbar-without-sidebar");
+    sessionStorage.setItem(STORAGE_TOGGLE_KEY, "un-toggled");
+    $(WRAPPER).trigger('sideBar::show');
+    $(WRAPPER).one("transitionend", function() {
+      $(WRAPPER).trigger('sideBar::shown');
+    })
+  }
+
+  function hide() {
+    $(SIDEBAR_CONTAINER).addClass('collapsed');
+    $(WRAPPER).css('paddingLeft', '0');
+    $('.navbar-secondary').addClass("navbar-without-sidebar");
+    sessionStorage.setItem(STORAGE_TOGGLE_KEY, "toggled");
+    $(WRAPPER).trigger('sideBar::hide');
+    $(WRAPPER).one("transitionend", function() {
+      $(WRAPPER).trigger('sideBar::hidden');
+    })
+  }
+
+  function toggle() {
+    var btn = $('#sidebar-arrow');
```

Comment:
Do we still need this code? You removed `#sidebar-arrow` and I think function `toggle` is not called from anywhere.